lib/WeBWorK/EquationCache.pm

```diff
@@ -37,6 +37,7 @@ extending to the end of the line is also ignored.
 use strict;
 use warnings;
 use Digest::MD5 qw(md5_hex);
+use Encode qw(encode_utf8 decode_utf8 decode);
```

Comment:
Why do you also include decode_utf8 and decode, when only encode_utf8 is used?

Reply:
Fair enough.  I wasn't sure what would be needed.